go/toolchains.rst

```diff
@@ -12,6 +12,9 @@ Go toolchains
 .. _register_toolchains: https://docs.bazel.build/versions/master/skylark/lib/globals.html#register_toolchains
 .. _compilation modes: modes.rst#compilation-modes
 .. _go assembly: https://golang.org/doc/asm
+.. _GoLibrary: providers.rst#GoLibrary
```

Comment:
I have no idea why, but GitHub is just showing the raw file instead of rendering it.